### PR TITLE
Run test suite via tox + Travis CI

### LINKED ISSUE #612: Problem: tox is still not running test suite

### PR DESCRIPTION
**Pull request by sevein**
- Added tox envs: `py27`, `py36`, `py27-flake8`, `py36-flake8`.
- Execute all envs in Travis CI.

This closes #612.

### CODE REVIEW

**Comment by helenst** [comment]
Looks good in terms of actually running the tests. Do we want to fix them all before merging this?

**Comment by sevein** [comment]
> Do we want to fix them all before merging this?

Yes, I'll give it a try today. It shouldn't be too hard but I'm surprised they broke. Thanks!

File: tox.ini [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+[testenv:archivematica]
+skip_install = True
+setenv =
+    PYTHONPATH = src/archivematicaCommon/lib:src/dashboard/src
 commands =
-    flake8 .
+    pip install -r src/archivematicaCommon/requirements/test.txt
+    pip install -r src/MCPClient/requirements/test.txt
+    pip install -r src/dashboard/src/requirements/test.txt
+    py.test {posargs}
```
Comment (sevein) [inline tox.ini]: `{posargs}` is used here so we can pass arguments to pytest, e.g.:

- `tox -e archivematica -- -x`
- `tox -e archivematica -- src/dashboard`
- [etc...](https://docs.pytest.org/en/latest/usage.html)

**Comment by sevein** [comment]
@jrwdunham @helenst, tests fixed! 😅

**Review by helenst — APPROVED** [review]
Nice work @sevein !

**Review by jrwdunham — APPROVED** [review]
Yay! Happy day.

**Pull request merged**